acceptance/ui/features/applications.feature

```diff
@@ -0,0 +1,95 @@
+@apps
+Feature: Application Management
+  In order to use Control Center
+  As a CC admin user
+  I want to manage applications
+
+  @login-required
+  Scenario: View default applications page
+    When I am on the applications page
+    Then I should be on the applications page
```

Comment:
I think this scenario should be similar to the first scenarios for Hosts and Pools where there are a set of "I should see" steps to verify the basic table contents.